samples/client/petstore/python-asyncio/petstore_api/api/pet_api.py

```diff
@@ -674,6 +688,12 @@ def get_pet_by_id_with_http_info(self, pet_id, **kwargs):  # noqa: E501
 
         # Authentication setting
         auth_settings = ['api_key']  # noqa: E501
+        
+        response_types_map={
+            200: "Pet",
+            400: "",
```

Comment:
Should these empty mappings be omitted?

Reply:
Yes it Should be empty 
````
"400": {
"description": "Invalid status value"
}
````
https://petstore.swagger.io/v2/swagger.json

Reply:
So then should the value be None for those cases?

Reply:
@spacether
done